cdc/sink/mysql.go

```diff
@@ -1005,3 +1005,92 @@ func buildColumnList(names []string) string {
 
 	return b.String()
 }
+
+//NewSyncpointSinklink create a sink to record the syncpoint map in downstream DB for every changefeed
+func NewSyncpointSinklink(ctx context.Context, info *model.ChangeFeedInfo, id string) (*sql.DB, error) {
+	var syncDB *sql.DB
+	// parse sinkURI as a URI
+	sinkURI, err := url.Parse(info.SinkURI)
```

Comment:
Can we reuse the code for connecting to DB?

Reply:
The new code has reused MySQLSink to sink syncpoint record instead of  using raw sql.DB.